pyproject.toml

```diff
@@ -142,3 +142,18 @@ line-length = 160
 
 [tool.ruff.lint]
 ignore = ["F403", "F405", "E402"]
+
+[tool.pixi.workspace]
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "osx-64", "linux-64", "win-64"]
+
+[tool.pixi.environments]
+default = { solve-group = "default" }
+dask = { features = ["dask"], solve-group = "default" }
+dev = { features = ["dev"], solve-group = "default" }
+parsl = { features = ["parsl"], solve-group = "default" }
+rucio = { features = ["rucio"], solve-group = "default" }
+triton = { features = ["triton"], solve-group = "default" }
+xrootd = { features = ["xrootd"], solve-group = "default" }
```

Comment:
So this works, but this is also going to get the _Python package_ version of all of these. If you're using Pixi and want to also leverage the conda package versions of your tools for different environment setups then you need to do some duplication work like I started to do in https://github.com/scikit-hep/pyhf/pull/2638.